lib/order_process.rb

```diff
@@ -0,0 +1,47 @@
+require_relative 'order'
+require_relative 'menu'
+require 'twilio-ruby'
+require_relative 'send_sms'
+
+class OrderProcess
+
+  def initialize
+    @restaurant_name = Menu.new.take_away_name
+    @order = Order.new
+  end
+
+  def show_menu
```

Comment:
Does the `OrderProcess` class need this method? Consider what actions belong to which class and only have the actions as methods inside of its owning class